jet/jet.go

```diff
@@ -150,41 +176,32 @@ func (e *Engine) Load() error {
 		if ext != e.extension {
 			return nil
 		}
-		// Get the relative file path
 		// ./views/html/index.tmpl -> index.tmpl
 		rel, err := filepath.Rel(e.directory, path)
 		if err != nil {
 			return err
 		}
-		// Reverse slashes '\' -> '/' and
-		// partials\footer.tmpl -> partials/footer.tmpl
-		name := filepath.ToSlash(rel)
-		// Remove ext from name 'index.tmpl' -> 'index'
-		name = strings.TrimSuffix(name, e.extension)
-		// name = strings.Replace(name, e.extension, "", -1)
+		name := strings.TrimSuffix(rel, e.extension)
 		// Read the file
 		// #gosec G304
 		buf, err := utils.ReadFile(path, e.fileSystem)
 		if err != nil {
 			return err
 		}
+
+		l.Set(name, string(buf))
 		// Create new template associated with the current one
 		// This enable use to invoke other templates {{ template .. }}
-		if _, err = e.Templates.LoadTemplate(name, string(buf)); err != nil {
-			return err
-		}
-		// Debugging
-		if e.debug {
```

Comment:
can you please leave that in

Reply:
Leave the print statement in when debugging is enabled?

Reply:
exactly, the debug currently only activates the dev-mode and there you have no information about the rendered templates(i tested it), think that is quite useful

Reply:
I'll fix it later today or tomorrow, np

Reply:
fixed